contracts/Migrations.sol

```diff
@@ -0,0 +1,25 @@
+pragma solidity ^0.4.11;
+
+
+contract Migrations {
+    address public owner;
+    uint public lastCompletedMigration;
+
+    modifier restricted() {
+        require(msg.sender == owner);
+        _;
```

Comment:
What's this?

Reply:
This is a _modifier_. When you use a modifier in a function, the code of the target function will be replaced with the modifier template hydrated with the target function. That's a common pattern to make modular contracts, see http://solidity.readthedocs.io/en/develop/common-patterns.html.

Reply:
I mean it's a _special symbol in a modifier_, see http://solidity.readthedocs.io/en/develop/contracts.html#function-modifiers.